test/e2e/driver/azuredisk_driver.go

```diff
@@ -77,6 +77,12 @@ func (d *azureDiskDriver) GetDynamicProvisionStorageClass(parameters map[string]
 		}
 	}
 
+	if strings.EqualFold(os.Getenv("AZURE_CLOUD_NAME"), "AZURESTACKCLOUD") {
```

Comment:
is it still necessary since in `controllerserver.go`, you already default the skuName to `Standard_LRS`

Reply:
yes, it is still necessary because in the migration test the in-tree provisioner does not have similar logic. Without this change, unacceptable sku still got to be sent to in-tree provisioner.